README.md

```diff
@@ -1,3 +1,123 @@
 # konan-housing-model
 
-A Konan-ready ML model, packaged and distributed as a Docker image, to estimate the sale price of house based on the popular Ames Housing Dataset
+[Konan](https://konan.ai) is your one-stop platform to bring your AI models to life. Typically, one would be required to do some tedious boring work in order to prepare the machine learning or data science model for deployment. Not with us! With **Konan**, gone are the days of worrying about a model's *dependencies*, battling with *containerization* or obsessively hand-holding the *infrastructure*. This repo will guide you in preparing your machine learning model in as seamless a manner as possible! Of course, don't forget to also checkout **Konan's** rich [documentation](https://docs.konan.ai) for additional info.
+
+This repository contains the source code and tooling necessarily to generate the **Konan**-compatible `konan-housing-model` docker image. Konan-ready ML model, packaged and distributed as a Docker image, to estimate the sale price of house based on the popular [Ames Housing Dataset](https://www.kaggle.com/datasets/prevek18/ames-housing-dataset) and the Jupyter [notebook](https://www.kaggle.com/code/jesucristo/1-house-prices-solution-top-1/notebook) titled *#1 House Prices Solution [top 1%]*, generously provided to the public by [Nanashi](https://www.kaggle.com/jesucristo).
+
+You can also find the built image and its tags on [Dockerhub](https://hub.docker.com/r/konanai/konan-housing-model) directly.
```

Comment:
I would add the docker instructions in the usage sections for those who want to only use the image